posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updated baseline standards for Windows targets"
+author: Chris Denton on behalf of the Compiler Team
+---
+
+The minimum requirements for Tier 1 toolchains targeting Windows will increase with the 1.77 release (scheduled for March 21, 2024).
+Windows 10 will now be the minimum supported version for the `*-pc-windows-*` targets.
+These requirements apply both to the Rust toolchain itself and to binaries produced by Rust.
+
+Two new targets have been added with Windows 7 as their baseline: `x86_64-win7-windows-msvc` and `i686-win7-windows-msvc`.
+They are starting as Tier 3 targets, meaning that the Rust codebase has support for them but we don't build or test them automatically.
+Therefore binaries aren't distributed via rustup.
+
+## Affected targets
+
+- `x86_64-pc-windows-msvc`
+- `i686-pc-windows-msvc`
+- `x86_64-pc-windows-gnu`
+- `i686-pc-windows-gnu`
```

Comment:
I believe the `*-windows-gnullvm` targets are also affected right? Or are we only listing Tier 1 targets here?

```suggestion
- `i686-pc-windows-gnu`
- `x86_64-pc-windows-gnullvm`
- `i686-pc-windows-gnullvm`
```